Hogajama/hogajama-angular-frontend/src/main/ionic/src/app/modules/shared/rule-dialog/rule-dialog.component.ts

```diff
@@ -0,0 +1,41 @@
+import {Component, Inject, OnInit} from '@angular/core';
+import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material';
+import {HogaramaBackendService} from '../../../services/HogaramaBackendService/hogarama-backend.service';
+import {Rule} from '../../../shared/models/Rule';
+
+@Component({
+  selector: 'app-rule-dialog',
+  templateUrl: './rule-dialog.component.html',
+  styleUrls: ['./rule-dialog.component.scss'],
+})
+export class RuleDialogComponent implements OnInit {
+
+  public rule: Rule;
+  public isEditAction = false;
+
+  constructor(
+      public dialogRef: MatDialogRef<RuleDialogComponent>,
+      @Inject(MAT_DIALOG_DATA) public data: Rule,
+      private backend: HogaramaBackendService
+  ) {
+    if (data != null) {
+      this.isEditAction = true;
+      this.rule = data;
+    } else {
+      this.rule = {};
+    }
+  }
+
+  ngOnInit() {}
+
+  async save() {
+    console.log(this.rule);
```

Comment:
Evtl. aus Versehen drin gelassen?

Edit: Ich sehe gerade, dass in anderen Komponenten auch sehr viele `console.log(...)`'s ohne Beschreibung drinnen sind. Sollten wir eventuell mal alle rauslöschen, oder auf einen custom logger umsteigen, der nur im debug mode läuft.